src/test/java/teammates/test/cases/browsertests/InstructorCourseEditPageUiTest.java

```diff
@@ -577,6 +577,48 @@ private void testEditInstructorAction() throws Exception {
                 sectionToCheck, sessionToCheck, InstructorCourseEditPage.SESSION_MODIFY_RESPONSES));
 
         courseEditPage.selectRoleForInstructor(editInstructorIndex, "Co-owner");
+
+        ______TS("Verify that one instructor must be visible to students");
+        // Need to prepare for test by setting all instructors as not displayed to students
+        courseEditPage.editInstructor(1, "New name", "new_email@email.tmt",
+                                      false, "Instructor", "Custom");
+        courseEditPage.editInstructor(2, "New name", "InsCrsEdit.instructor@gmail.tmt",
+                false, "New display name", "Co-owner");
+        courseEditPage.editInstructor(3, "New name", "InsCrsEdit.instructor@gmail.tmt",
+                false, "Instructor", "Co-owner");
+        courseEditPage.editInstructor(4, "Teammates New Instructor", "InsCrsEdit.Helper@gmail.tmt",
+                false, "Instructor", "Custom");
+        courseEditPage.editInstructor(5, "Teammates New Instructor", "InsCrsEdit.newInstr@gmail.tmt",
+                false, "Instructor", "Custom");
+        courseEditPage.editInstructor(6, "Teammates Observer", "InsCrsEdit.observer@gmail.tmt",
+                false, "Instructor", "Observer");
+        courseEditPage.editInstructor(7, "Teammates Test", "InsCrsEdit.test@gmail.tmt",
+                false, "Instructor", "Co-owner");
+        courseEditPage.editInstructor(8, "Teammates Do Not Save", "InsCrsEdit.tutor@gmail.tmt",
+                false, "Instructor", "Tutor");
+        /* Test to verify that changes made to instructor are not saved if this is the only instructor displayed to students
+        and you try to uncheck the box for "Display to students as:"
+        The information in "verifyInstructorDetails" method below comes from InstructorCourseEditPageUiTest.json,
+        googleId: InsCrsEdit.tutor
+         */
```

Comment:
For comments spanning multiple lines within a method, use multiple single-line comments instead ie.

```java
// a comment on what this code does
// spanning multiple lines
```

This reduces the complications involved with commenting out multiple lines of code during development later on (should that case happen).